service/src/test/java/org/mskcc/smile/service/SampleServiceTest.java

```diff
@@ -323,4 +323,43 @@ public void testFindSamplesByDate() throws Exception {
                 sampleRepository.findLatestSampleMetadataBySmileId(updatedSample.getSmileSampleId());
         Assertions.assertThat(latestMetadata.getCmoSampleName()).isEqualTo("C-LATESTLABEL-T02-d002");
     }
+
+    /**
+     * Tests that sample can be found by a valid igoId
+     * @throws Exception
+     */
+    @Test
+    public void testFindSampleByIgoId() throws Exception {
+        String igoId = "MOCKREQUEST1_B_2";
+
+        SmileSample sample = sampleService.getSampleByInputId(igoId);
+        Assertions.assertThat(sample).isNotNull();
+    }
+
+    /**
+     * Tests that sample can be found by a valid uuid
+     * @throws Exception
+     */
+    @Test
+    public void testFindSampleByUuid() throws Exception {
+        String igoId = "MOCKREQUEST1_B_2";
+
+        SmileSample sample = sampleService.getSampleByInputId(igoId);
+        String uuid = sample.getSmileSampleId().toString();
+        SmileSample sampleByUuid = sampleService.getSampleByInputId(uuid);
+
+        Assertions.assertThat(sampleByUuid).isNotNull();
+    }
+
+    /**
+     * Tests that sample can not be found by a invalid inputId
+     * @throws Exception
+     */
+    @Test
+    public void testFindSampleByInvalidInputId() throws Exception {
+        String inputId = "invalidInput";
+
+        SmileSample sample = sampleService.getSampleByInputId(inputId);
+        Assertions.assertThat(sample).isNull();
+    }
```

Comment:
we can remove these tests if you expand the test above (see comment for more info)

we want to do more than just check if the returned object is not null